mobile/src/main/java/rs/readahead/washington/mobile/domain/entity/resources/ListResourceResult.kt

```diff
@@ -0,0 +1,13 @@
+package rs.readahead.washington.mobile.domain.entity.resources
+
+import rs.readahead.washington.mobile.data.entity.reports.ProjectSlugResourceResponse
+
+class ListResourceResult {
+    var errors: List<Throwable> = ArrayList()
```

Comment:
covert this into a data class 
```
  data class ListResourceResult(
    var errors: List<Throwable> = emptyList(),
    var slugs: List<ProjectSlugResourceResponse> = emptyList()
)
```

Reply:
Consider using `emptyList()` instead of ` ArrayList()` for the default values. This ensures that the default lists are immutable.